x-pack/plugins/session_view/public/components/session_view/index.tsx

```diff
@@ -67,9 +74,11 @@ export const SessionView = ({
     let source = '';
     // append 'app' details (which telemtry source is this from?)
     if (index === CLOUD_DEFEND_INDEX) {
-      source += 'cloud-defend';
+      source += CLOUD_DEFEND_DATA_SOURCE;
     } else if (index === ENDPOINT_INDEX) {
-      source += 'endpoint';
+      source += ELASTIC_DEFEND_DATA_SOURCE;
+    } else if (index === AUDITBEAT_INDEX) {
+      source += AUDITBEAT_DATA_SOURCE;
     } else {
       // any telemetry producers setting process.entry_leader.entity_id will cause sessionview action to appear in timeline tables.
       source += 'unknown';
```

Comment:
Yes I guess that's a nice suggestion, the original implementation suggests `source` can append strings, but from what I can see it will always be only one value